Sources/NIO/AsyncAwaitSupport.swift

```diff
@@ -0,0 +1,92 @@
+#if compiler(>=5.4)
+extension EventLoopFuture {
+    public func get() async throws -> Value {
```

Comment:
I'm a bit concerned about the name of this method: it's too generic and wide. I would go for something more specific, like `async()`, in which case it would make more sense (awaiting async thing), however, I'm not sure if it's allowed by compiler anymore.

I think you could also add an extension for `ELF<Void>` with non-returning async method, which would make even more sense with `async()` naming (rather than `get()`). Or alternatively make this method `@discardableResult`.

Reply:
`get()` would be much more consistent with the rest of the ecosystem. For example, [in the structured concurrency proposal](https://github.com/DougGregor/swift-evolution/blob/structured-concurrency/proposals/nnnn-structured-concurrency.md#task-handles) task handles have an almost identical `get() async throws -> Success` function on them.

Reply:
We use `get()` everywhere in the language so having this do the same is the right way; however ugly the method name is 😉

Reply:
+1 on `@discardableResult` though

Reply:
Yeah, `Result` from stdlib has this too: `func get() throws -> Success`.